app/src/main/java/com/example/tasky/common/presentation/SimpleButton.kt

```diff
@@ -34,6 +41,35 @@ fun SimpleButton(
     }
 }
 
+@Composable
+fun PillButton(
+    buttonName: String,
+    onClick: () -> Unit,
+    isSelected: Boolean = false
+) {
+    val containerColor = if (isSelected) Color.Black else colorResource(id = R.color.reminder_gray)
+    val contentColor = if (isSelected) Color.White else Color.Black
+
+    androidx.compose.material3.Button(
+        onClick = onClick,
+        modifier = Modifier
+            .padding(start = 8.dp, end = 8.dp)
+            .width(110.dp)
+            .height(35.dp),
```

Comment:
Does every button in the app have exactly these dimensions on all different screen sizes? I'd recommend to give composables a relative size by default, unless it's really the same across the app. For example, here you could specify a default padding to avoid making this button too cramped, but leave the rest up to the calling composable by passing a custom modifier to this one.